### PR TITLE
New pages and functionality. Added article and video .js files as well.

### PR DESCRIPTION
**Pull request by hfakhreddine86**
@nscox @abonnema Please review all the changes and let me know what you think. If there is any thing you would like to change please comment in details. You will see that I went in a different direction for the functionality of the page. We will no longer have dietary classification pages separately. All the recipes and ingredients and restaurant info will be filtered and produced on the index.html page. The other pages are now going to display articles pulled from google news API and how to videos from youtube API. The about.html page will display our BIO and a profile picture.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,7 +75,7 @@
                     </div>
                     <div class="divider"></div>
                     <div class="section">
-                        <div>Add any dietary restrictions or deficiencies</div>
+                        <div>Add any dietary restrictions or deficiencies:</div>
                         <div class="input-field col l6">
                             <select multiple>
                                 <option value="1">Halal Only</option>
```
Comment (abonnema) [inline index.html]: We may only have to put Kosher only and Crustacean-free for the dietary restrictions according to the Edamam API

File: index.html [hunk 2]
```diff
@@ -56,12 +56,12 @@
...
-            <div class="col s12 m8 l9">
+            <div class="col s12 m12 l12">
                 <section>
-                    <div class="welcome">Welcome! eatSentialSmart is a tool that helps you find food!</div>
+                    <h3>Food Finder</h3>
                     <div class="divider"></div>
                     <div class="section">
-                        <div>Please select your dietary classification</div>
+                        <div>Please select your dietary classification:</div>
                         <input class="with-gap" name="classification" type="radio" id="omnivore" />
```
Comment (abonnema) [inline index.html]: Match up radio button with proper health label today. Look at NYT Documentation

Comment (hfakhreddine86) [reply index.html]: Could you elaborate on this please... For label you mean class?

File: index.html (hunk 1 above)
Comment (hfakhreddine86) [reply index.html]: What are some other options as to what restrictions there are?

**Pull request merged**